603-spring-web-smallrye-openapi/src/test/java/com/baeldung/springbootbootstrap/SpringBootBootstrapLiveTest.java

```diff
@@ -0,0 +1,135 @@
+package com.baeldung.springbootbootstrap;
+
+import static org.apache.commons.lang3.RandomStringUtils.randomAlphabetic;
+import static org.apache.commons.lang3.RandomStringUtils.randomNumeric;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+
+import com.baeldung.springbootbootstrap.persistence.model.Book;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+
+@QuarkusTest
+public class SpringBootBootstrapLiveTest {
```

Comment:
I didn't get the point of naming this test class `SpringBootBootstrapLiveTest`. What about BookResourceTest or BookRestResourceTest?

Reply:
I went on with `BookResourceTest`.